packages/webapp/src/components/Map/LineMapBoxes/index.jsx

```diff
@@ -58,7 +62,10 @@ export default function PureLineBox({
 
   useEffect(() => {
     trigger();
-    updateWidth((widthValue || 0) + (bufferWidthValue || 0));
+    updateWidth(
```

Comment:
On my end, the max value seems to be working without this code. I believe this can be reverted to `updateWidth((widthValue || 0) + (bufferWidthValue || 0));`

Reply:
Though I guess this code would prevent large values from actually showing on google maps.

Reply:
Nevermind, this looks good to me.